.scalafmt.conf

```diff
@@ -0,0 +1,12 @@
+version = 3.2.2
+runner.dialect = scala213
+maxColumn = 100
+newlines.source = keep
+newlines.alwaysBeforeElseAfterCurlyIf = true
+indentOperator.excludeRegex = "^$"
+danglingParentheses.defnSite = false
+danglingParentheses.callSite = false
+rewrite.trailingCommas.style = keep
+binPack.literalsExclude = []
+binPack.literalsIncludeSimpleExpr = true
+docstrings.style = Asterisk
```

Comment:
Maybe just go with the default settings? Not sure where these are coming from. :)